mkdocs.yml

```diff
@@ -1,13 +1,17 @@
 site_name: "Pi-hole documentation"
 site_url: https://docs.pi-hole.net
-repo_url: "https://github.com/pi-hole/pi-hole/"
+repo_url: "https://github.com/pi-hole/pi-hole"
 edit_uri: ""
-copyright: 'Copyright &copy; 2018 Pi-hole LLC'
+copyright: 'Copyright &copy; 2019 Pi-hole LLC'
 remote_branch: gh-pages
 theme:
   name: 'material'
   favicon: 'images/favicon.png'
+  logo: 'images/logo.svg'
```

Comment:
Took me a while to see where this was even placed on the site, subtle - but effective!